### DISCUSSION TITLE
Up/Down/Left/Right not working on Firestick4K

### DISCUSSION THREAD

**Discussion by TurboTronix** [PRProd/HA-Firemote#199] [Q&A] [answered]
Can't seem to get the up/down/left/right circle to work. I can get into the various apps using their buttons at the bottom but none of the navigation buttons work (home button does work).

**Comment by PRProd** [comment, marked as answer]
Hello @TurboTronix!

Have you had a chance to check out the [FAQ](https://github.com/PRProd/HA-Firemote#faq) section in the readme file?  I suggest reading the section titled [Why don't any of the buttons on the Firemote work at all?](https://github.com/PRProd/HA-Firemote#why-dont-any-of-the-buttons-on-the-firemote-work-at-all).

After reading and following the advice there, please let me know if you need any further help.

**Reply by TurboTronix** [reply]
Yes that worked thanks
